openedx/core/djangoapps/schedules/tests/test_signals.py

```diff
@@ -107,6 +107,18 @@ def test_create_schedule_hold_backs(
         else:
             self.assert_schedule_not_created()
             mock_track.assert_called_once()
+            self.assertEquals(mock_track.call_args[1].get('event'), 'edx.bi.schedule.suppressed')
+
+    @patch('openedx.core.djangoapps.schedules.signals.log.exception')
+    @patch('openedx.core.djangoapps.schedules.signals.Schedule.objects.create')
+    def test_create_schedule_error(self, mock_create_schedule, mock_log, mock_get_current_site):
+        site = SiteFactory.create()
+        mock_get_current_site.return_value = site
+        ScheduleConfigFactory.create(site=site)
+        mock_create_schedule.side_effect = ValueError('Fake error')
+        self.assert_schedule_not_created()
+        mock_log.assert_called_once()
+        assert 'Encountered error in creating a Schedule for CourseEnrollment' in mock_log.call_args[0][0]
```

Comment:
Use `self.assertIn`.

Reply:
I thought that now that we are using pytest we would prefer `assert <expression>` versus the assert functions like `self.assertIn`. Personally I prefer using the plain `assert` because then I don't have to remember which assert function to use.

Reply:
I was erring on the side of consistency throughout the codebase unless a decision was made to change this standard.  @jmbowman @jzoldak @nedbat  any opinions on this matter?

Reply:
The plain "assert" generally produces better output with pytest, in addition to being easier to remember how to use.  I'm 👍 on gradually switching everything over to it, although I'll grant that inconsistency within a single test method can be confusing.

Reply:
I've written some pytest-style tests (no classes, just functions) in edx-platform already.

Reply:
Great.  Thanks for the feedback all!  In that case, @thallada if it's not too much work, would you be able to update the other asserts in this file to use the pytest version - per @jmbowman's point about consistency within this file?  There aren't that many right now - and before inconsistency causes confusion to future readers of this module.